packages/plans-grid-next/src/hooks/use-manage-tooltip-toggle.ts

```diff
@@ -14,7 +14,7 @@ export function useManageTooltipToggle(): [ string, Dispatch< SetStateAction< st
 		// Since the plan logos are an SVG, we are checking the event target's parent classes to determine the tooltip container.
 		const closeAllTooltips = ( event: TouchEvent ) => {
 			const eventTarget = event.target as Element;
-			if ( ! eventTarget.closest( '.plans-2023-tooltip__hover-area-container' ) ) {
+			if ( ! eventTarget.closest( '.plans-next-tooltip__hover-area-container' ) ) {
```

Comment:
Should this be `.plans-grid-next-tooltip__hover-area-container` instead (with `-grid-` in it)?

Curious about what sort of breakage this was leading to (and how neither unit tests nor manual testing caught anything around it) 🤔

^ This may reflect back on @jeyip 's comment about "tests with value" (assuming this bit of logic is critical for the tooltips)

Reply:
Great catch, thanks!

This is a really good example where we can/should rely on unit tests (fast, predictable, automatic) and not manual testing (time consuming, error prone, etc). 

The unit tests [I have now added](https://github.com/Automattic/wp-calypso/pull/93518/commits/5cfeb08e6b93155295e83b1f0d19c2bb88b13a4e) for `useManageTooltipToggle` will break if a future refactor changes the classname without updating all instances.

Reply:
Since these tests you are adding are really covering a set of minimal operations over a fairly minimal component, that I doubt will ever change in any significant way (outside of being deprecated), would just adding a snapshot test suffice for the whole lot?

Reply:
Replied here, but obviously referring to tooltip comp